auth-api/src/auth_api/models/user.py

```diff
@@ -214,11 +214,13 @@ def _get_type(cls, user_from_context: UserContext) -> str:
         """Return type of the user from the token info."""
         user_type: str = None
         if user_from_context.roles:
-            if Role.ANONYMOUS_USER.value in user_from_context.roles:
+            if Role.ANONYMOUS_USER.value in user_from_context.roles \
```

Comment:
first time login call wont have the roles, so adding the login source as a fall back.